### PR TITLE
feat (JS/FE-2023Q4): Translate HTML Forms module

### PR DESCRIPTION
**Pull request by haslie22**
- [x] Translate the entire content of the ["HTML Forms & Validation" module](https://github.com/rolling-scopes-school/tasks/tree/master/stage1/modules/html-form) to English.
- [x] Review the current materials, including tutorials, examples, and exercises, for relevance and accuracy.

### CODE REVIEW

File: stage1/modules/html-form/README.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-1. Вам необходимо пройти тест "HTML Form" в RS APP > Auto Test
+## Practice 
+1. You need to take the "HTML Form" test in the RS APP > Auto Test
 
-## Дополнительные материалы
-1. https://learn.javascript.ru/forms-controls
-2. [Запись вебинара RS School](https://www.youtube.com/watch?v=PhRVJC0kBGE)
+## Additional materials
+1. [Forms, controls](https://javascript.info/forms-controls)
+2. [RS School recorded webinar](https://www.youtube.com/watch?v=PhRVJC0kBGE)
```
Comment (valerydluski) [inline stage1/modules/html-form/README.md:21]: `2. [RS School recorded webinar (RU)](https://www.youtube.com/watch?v=PhRVJC0kBGE)`

**Comment by valerydluski** [comment]
Could you please add this link? https://web.dev/learn/forms/

File: stage1/modules/html-form/README.md (hunk 1 above)
Comment (haslie22) [reply stage1/modules/html-form/README.md]: Done 👌

Comment (haslie22) [reply stage1/modules/html-form/README.md]: Also fixed module duration time, added 2 hours for https://web.dev/learn/forms/

**Pull request merged**